entries/s/simplelogin.io.json

```
@@ -0,0 +1,12 @@
+{
+  "Simple Login": {
```

The correct spelling seems to be as one word.
```suggestion
  "SimpleLogin": {
```